tests/core/repository/mocks.py

```diff
@@ -39,9 +38,9 @@ def __init__(self, id: str, name: str, version: Optional[str] = None) -> None:
         self.id = id
         self.name = name
         if version:
-            self._version = version
+            self.version = version
```

Comment:
```suggestion
            self._version = version
```
I believe this is a conflict with merged PR https://github.com/Avaiga/taipy-core/pull/663